src/common/args.cpp

```diff
@@ -112,18 +144,75 @@ std::optional<common::SettingsValue> InterpretValue(const KeyInfo& key, const st
             error = strprintf("Negating of -%s is meaningless and therefore forbidden", key.name);
             return std::nullopt;
         }
+        if (TypedArg(flags)) {
+            if (value && *value != "1") {
+                error = strprintf("Can not negate -%s at the same time as setting value '%s'.", key.name, *value);
```

Comment:
Small adjustment to clarify that the specific value being set is not important.
```suggestion
                error = strprintf("Can not negate -%s at the same time as setting a value ('%s').", key.name, *value);
```